appveyor-release.yml

```diff
@@ -0,0 +1,61 @@
+version: 1.0.{build}
+image:
+  - Visual Studio 2017
+  - Ubuntu
+environment:
+  matrix:
+    - ARCH: amd64
+    - ARCH: arm 
+    - ARCH: arm64
+    - ARCH: ppc64le
+    - ARCH: s390x
+  DOCKER_USER:
+    secure: JapmC7j5F0mY3j/MVzU+Cw==
+  DOCKER_PASS:
+    secure: QGlCLNWzPD0HL8ipkohVic45/yU3bVOdjn0IiV6NnSQ=
+  PORTAINER_VERSION: "1.19.2"
```

Comment:
Why do we need to specify the `PORTAINER_VERSION` here?

Reply:
As discussed, it is to control the Portainer version information for container images from a single point

Reply:
All good